commit fbebfbe29d1928a36fe5643853b5a02f13f7b07c
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Sun Jul 23 22:21:38 2017 +0200

    Cleanup unused dependencies
    
    Test Plan: Builds fine.
    
    Reviewers: #kde_pim, mlaurent
    
    Reviewed By: #kde_pim, mlaurent
    
    Subscribers: mlaurent
    
    Tags: #kde_pim
    
    Differential Revision: https://phabricator.kde.org/D6859

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dca7fe2..d5f41fc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,23 +47,18 @@ find_package(KF5DBusAddons ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5I18n ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5IconThemes ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5ItemModels ${KF5_VERSION} CONFIG REQUIRED)
-find_package(KF5ItemViews ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5JobWidgets ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5KIO ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5NewStuff ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5Service ${KF5_VERSION} CONFIG REQUIRED)
-find_package(KF5Wallet ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5WidgetsAddons ${KF5_VERSION} CONFIG REQUIRED)
-find_package(KF5WindowSystem ${KF5_VERSION} CONFIG REQUIRED)
 find_package(KF5XmlGui ${KF5_VERSION} CONFIG REQUIRED)
 
 find_package(KF5Akonadi ${AKONADI_VERSION} CONFIG REQUIRED)
 find_package(KF5AkonadiContact ${AKONADICONTACT_LIB_VERSION} CONFIG REQUIRED)
-find_package(KF5AkonadiMime ${AKONADIMIME_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5Contacts ${KCONTACTS_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5IMAP ${KIMAP_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5LibkdepimAkonadi ${LIBKDEPIM_LIB_VERSION} CONFIG REQUIRED)
-find_package(KF5Mime ${KMIME_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5PimTextEdit ${KPIMTEXTEDIT_LIB_VERSION} CONFIG REQUIRED)
 find_package(Grantlee5 "5.1" CONFIG REQUIRED)
 
@@ -82,6 +76,7 @@ find_package(Xsltproc)
 set_package_properties(Xsltproc PROPERTIES DESCRIPTION "XSLT processor from libxslt" TYPE REQUIRED PURPOSE "Required to generate D-Bus interfaces for all Akonadi resources.")
 
 if(BUILD_TESTING)
+   find_package(KF5Mime ${KMIME_LIB_VERSION} CONFIG REQUIRED)
    add_definitions(-DBUILD_TESTING)
 endif(BUILD_TESTING)
 
